api-specificatie/openapi.yaml

```diff
@@ -579,24 +563,15 @@ components:
         - Ligplaats
 #JBO: Uit https://github.com/lvbag/BAG-API/blob/master/Technische%20specificatie/Yaml's/20191209-1-lvbag-individuele-bevragingen-20180601/lvbag/imbag/v20180601/ligplaats.yaml (+aanpassingen)
     Ligplaats:
```

Comment:
Moet hier niet het type LigplaatsIO gebruikt worden? Als de indicatie InOnderzoek=true in Adres, wil een gebruiker dan weten welke attributen er inonderzoek zijn? Als we dat direct meeleveren, hoeft er geen vervolgvraag gesteld te worden.

Reply:
Ik heb LigplaatsIO niet gebruikt omdat ik het element "voorkomen" uit de specificatie heb verwijderd. Overigens heb ik  "inonderzoek" wel meegenomen in de definitie van van het component "Ligplaats". Dus het is wel zo gespecificeerd zoals jij voorsteld, Inonderzoek komt meteen mee als daar spraken van is.